core/bug_api.php

```diff
@@ -783,6 +971,36 @@ function update( $p_update_extended = false, $p_bypass_mail = false ) {
 
 		return true;
 	}
+	
+	/**
+	 * Increase votes/count for bug by 1 and return new value
+	 * @return integer
+	 * @access public
+	 */
+	function vote() {
+		$c_bug_id = $this->id;
+		
+		db_param_push();
+		$t_query = 'UPDATE {bug}
+						SET votes = votes + 1
+						WHERE id=' . db_param();
+		db_query( $t_query, array(
+			$c_bug_id ) );
+			
+		$c_votes = $this->votes;	
+			
+		db_param_push();
+		$t_query = 'SELECT votes FROM {bug}
+						WHERE id=' . db_param();
+		$t_result = db_query( $t_query, array(
+						$c_bug_id ) );
+			
+		$this->votes = db_result( $t_result );	
+
+		history_log_event_special( $c_bug_id, BUG_VOTED );
```

Comment:
Probably not. 

+1/-1 could be implemented as "BUG_UP_VOTED"/"BUG_DOWN_VOTED". Whereas an integer arg would suggest that you can up/down vote by more than 1 (e.g. +100).

Anyway, this can be modified later when real voting will be added.